docs/source/modin/supported/datetime_index_supported.rst

```diff
@@ -100,7 +100,7 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``day_name``                | P                               | ``locale``                       |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``mean``                    | N                               |                                  |                                                    |
+| ``mean``                    | Y                               |                                  |                                                    |
```

Comment:
they should be P right, some arguments are not supported?

Reply:
Only `ddof` for std is not supported, but I updated that now.